install/setup/sql/mysql/kylo/0.9.0/abandon-feed-jobs.sql

```diff
@@ -0,0 +1,22 @@
+USE kylo;
+
+DROP PROCEDURE IF EXISTS `abandon_feed_jobs`;
+
+DELIMITER $$
+CREATE PROCEDURE `abandon_feed_jobs`(in feed varchar(255), in exitMessage varchar(255))
+BEGIN
+
+UPDATE BATCH_JOB_EXECUTION
+  INNER JOIN BATCH_JOB_INSTANCE ON BATCH_JOB_INSTANCE.JOB_INSTANCE_ID = BATCH_JOB_EXECUTION.JOB_INSTANCE_ID
+SET BATCH_JOB_EXECUTION.STATUS = 'ABANDONED',
+  BATCH_JOB_EXECUTION.EXIT_MESSAGE = exitMessage
```

Comment:
Can you change this so it concats the previous exit messages with a new line.
`BATCH_JOB_EXECUTION.EXIT_MESSAGE = CONCAT_WS('\n',BATCH_JOB_EXECUTION.EXIT_MESSAGE,exitMessage)`